setup.py

```diff
@@ -7,7 +7,7 @@
 with open('README.md') as readme_file:
     readme = readme_file.read()
 
-requirements = ['webexpythonsdk==2.0.2', 'coloredlogs', 'websockets==11.0.3', 'backoff']
+requirements = ['webexpythonsdk>=2.0.3', 'coloredlogs', 'websockets==11.0.3', 'backoff']
```

Comment:
This should be done in a controlled fashion to avoid PRs suddenly failing builds and to give a consistent build environment. Thats why we pin the version in the code.

```suggestion
requirements = ['webexpythonsdk==2.0.3', 'coloredlogs', 'websockets==11.0.3', 'backoff']
```